### DISCUSSION TITLE
VoIPMediaSession not closing VideoSink

### DISCUSSION THREAD

**Discussion by julius-iden** [sipsorcery-org/sipsorcery#491] [Q&A]
Is there a reason for the VoIPMediaSession not closing the VideoSink? From what I see all other sources are closed. For the video sink only the eventhandlers unsubscribe.
Am I missing something?

**Comment by sipsorcery** [comment]
For a video sink there is nothing to close.

An audio sink is an actual device like a speaker. A video sink doesn't have an equivalent. Instead it simply supplies image frames to it's events.

**Reply by julius-iden** [reply]
So the CloseVideoSink in the interface is meant to be called by the application?
